ReactSkia/sdk/CurlNetworking.cpp

```diff
@@ -101,10 +103,16 @@ void CurlNetworking::processNetworkRequest(CURLM *curlMultiHandle) {
   int msgsLeft = 0;
 
   do {
-    res = curl_multi_perform(curlMultiHandle, &stillAlive);
+    {
+      std::lock_guard<std::mutex> lock(mutex_);
+      res = curl_multi_perform(curlMultiHandle, &stillAlive);
```

Comment:
we should use different name for mutex_ variable. Also it is not clear  about the purpose of mutex_ .  Is it not required everywhere curlMultihandle is used ?